src/Example.php

```diff
@@ -1,11 +1,18 @@
 <?php
 
+declare(strict_types=1);
+
 namespace App;
 
 class Example
 {
-    public function hello()
+    public function hello(): string
     {
         return 'hi';
     }
+
+    public function addition(int $numberOne, int $numberTwo): int
```

Comment:
I added this method just to demostrate in the test about how to use data provider to make test more efficient. It's important